src/app.js

```diff
@@ -5,11 +5,22 @@ env.config();
 // import libraries
 const express = require('express');
 const cors = require('cors');
+const { auth } = require('express-openid-connect');
 const app = express();
 
 // initialize database and schemas
 require('./model/db');
 
+// config info for auth
+const config = {
```

Comment:
After logging in, we should be redirected to ``localhost:3000`` (our website) instead of ``localhost:8081`` (our server). By doing this, the auth cookie will be stored on our website. 

We _might_ also need to configure the auth0 application to allow callbacks to localhost:3000 (not sure about this):
https://auth0.com/docs/quickstart/spa/vanillajs/01-login

Reply:
Resolved